lib/ember-cli/deploy/redis.rb

```diff
@@ -0,0 +1,82 @@
+require "active_support/core_ext/object/blank"
+require "ember-cli/deploy/page"
+require "redis"
+
+module EmberCLI
+  module Deploy
+    class Redis
+      def initialize(namespace:, index_html: nil)
```

Comment:
It would be nice to be able to inject the Redis client as an optional parameter. I am running on puma, and ended up running out of redis connections after a few page loads because the existing connection wasn't being closed. If I could pass in a redis client, I should be able to use a connection pool so I don't run into this issue.